lib/modules/dosomething/dosomething_campaign/includes/Campaign.php

```diff
@@ -363,6 +366,25 @@ protected function getMobileCommonsData() {
   }
 
 
+   /**
+   * Get the start and end dates for when campaign will be displayed on the mobile app if available.
+   * Dates formatted as ISO-8601 datetime.
+   *
+   * @return array
+   */
+
+    protected function getMobileAppDate() {
+      $timezone = new DateTimeZone('UTC');
```

Comment:
I think this line for the `$timezone` can probably just go inside the `dosomething_helpers_convert_date()` function :)